workspaces/orchestrator/app-config.yaml

```diff
@@ -35,6 +35,9 @@ backend:
     connection: ':memory:'
   cache:
     store: memory
+  # useful for debugging APIs via curl in dev env without requiring a auth:
```

Comment:
This can be achieved by https://backstage.io/docs/auth/service-to-service-auth#static-tokens
Very easy to set up and use.